sdv/metadata/anonymization.py

```diff
@@ -71,7 +71,7 @@ def _detect_provider_name(function_name):
         return 'BaseProvider'
 
     else:
-        return module[-1]
```

Comment:
Without the changes from RDT, does this introduce any new bugs or does it maintain the same functionality as the current release?